device-sdks/index.html

```diff
@@ -0,0 +1,2205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Astarte Device SDKs</title>
+
+    <link rel="stylesheet" href="../assets/dist/styles.css" />
+  </head>
+  <body onload="populateData('C')">
+    <section class="sdk-intro-container">
+      <h1 class="astarte-title">Astarte Device SDKs</h1>
+      <div class="sdk-content">
+        <h2 id="introduction" class="sdk-heading">
+          <a href="#introduction" class="hover-link"
+            ><span class="icon-link" aria-hidden="true"></span
+          ></a>
```

Comment:
This formatting option feel strange to me. Which prettier config are you using?
I would expect this to be formatted as:
```html
<a href="#introduction" class="hover-link">
  <span class="icon-link" aria-hidden="true" />
</a>
```

Reply:
I use default prettier config and i have correct this formatting.